game/world/worldobjects.cpp

```diff
@@ -799,7 +799,7 @@ Interactive *WorldObjects::availableMob(const Npc &pl, std::string_view dest) {
 
   float curDist=dist*dist;
   interactiveObj.find(pl.position(),dist,[&](Interactive& i){
-    if(i.isAvailable() && i.checkMobName(dest)) {
+    if((ignoreInUse || i.isAvailable()) && i.checkMobName(dest)) {
```

Comment:
Should `ignoreInUse==true` be the only behavior?

Reply:
This isn't needed anymore now with aiqueue not being leveraged.